doc/source/reference/security.rst

```diff
@@ -0,0 +1,61 @@
+NumPy security
+==============
+
+Security issues can be reported privately as described in the project README
+and when opening a `new issue on the issue tracker <https://github.com/numpy/numpy/issues/new/choose>`_.
+The `Python security reporting guidlines <https://www.python.org/dev/security/>`_
+are a good resource and its notes apply also to NumPy.
+
+NumPy's maintainers are not security experts.  However, we are conscientious
+about security and experts of both the NumPy codebase and how it's used.
+Please do notify us before creating security advisories against NumPy as
```

Comment:
I was just curious about this bit, `Please do notify us`. We cannot create a public issue or hit the mailing list with the risk of exposure, so what is the preferred way (like Tidelift directly)?

Reply:
The previous paragraph gives the appropriate instructions for reporting privately.